helpers/calculateCabbages.ts

```diff
@@ -20,13 +20,13 @@ export const getCabbageBreakdown = (
     });
 
     const cabbageBreakdown = {
-        eventCabbages: memberData.eventCabbages,
+        eventCabbages: memberData.eventCabbages || 0,
```

Comment:
Are these changes (the ` || 0`)'s related to the RSN display issue, or just additional error state handling?

No change needed, this is obviously an improvement, I am just curious if I am missing something.

Reply:
Yes! Additional error state handling! **Theoretically** this should never happen, but in case it somehow does, then the cabbages will not be totalized incorrectly, instead resulting in `NaN`